plugins/kubernetes-backend/config.d.ts

```diff
@@ -55,7 +56,8 @@ export interface Config {
               | 'google'
               | 'googleServiceAccount'
               | 'oidc'
-              | 'serviceAccount';
+              | 'serviceAccount'
+              | string;
```

Comment:
This will flatten the types here completely - `'enum' | string` usually just resolves to `string`. Maybe this doesn't need to be an enum at all?

Reply:
I understood this file does provide the autocompletion features also. So left it like this so the default values are still "completed". 

If not, the others can for sure be eliminated.

Reply:
In that case I will change it for sure!

Reply:
@aramissennyeydd thanks for the comment! 
Adapted the file now so resolved it.